homeassistant/components/fitbit/sensor.py

```diff
@@ -259,7 +283,7 @@ class FitbitSensorEntityDescription(SensorEntityDescription):
     ),
     FitbitSensorEntityDescription(
         key="activities/tracker/distance",
-        name="Tracker Distance",
+        translation_key="distance",
```

Comment:
```suggestion
```

Reply:
Thanks, makes sense. I also added a test for this one just to make sure it's exercised. Regenerated the snapshot too.